### DISCUSSION TITLE
Alter table setting (old format)

### DISCUSSION THREAD

**Discussion by rkozlo** [ClickHouse/ClickHouse#28454] [Q&A] [answered]
Hi, i have tables in old format MergeTree(date-column [, sampling_expression], (primary, key), index_granularity). I try modify settings on this table and it says me it is not supported on this table engine. Is it mergetree or not? How can i change it? I really need it.

**Comment by den-crane** [comment, marked as answer]
duplicate https://github.com/ClickHouse/ClickHouse/issues/28006#issuecomment-903734247
